docs/src/intro-js.md

```diff
@@ -138,7 +138,8 @@ module.exports = config;
 
 ```js js-flavor=ts
 // playwright.config.ts
-import { PlaywrightTestConfig, devices } from '@playwright/test';
+import type { PlaywrightTestConfig } from '@playwright/test';
```

Comment:
You can do `import { type PlaywrightTestConfig, devices }`

Reply:
@jsakamoto How do you feel about using inline notation instead?

Reply:
@pavelfeldman 
Yes, I feel that the inline notation is a bit smarter than my pull request. 👍
However, I just followed the code that the `npm init playwright` command generated.

Anyway, I strongly hope this pull request is accepted **rapidly**, even if my pull request is not the best.
Because there might be some people who are wasting time and energy due to that syntax error at this moment.
I was one of those people.

After doing that, it would be better to keep improving the documents to be smarter by using inline notation.